flair/trie/replication.go

```diff
@@ -65,21 +66,24 @@ func (r *Replicator) Sequential(key string, f ReplicatedFunc) error {
 // This facilitates the BatchReadBlobs endpoint that basically never returns an 'actual' error
 // because they're all inline.
 func (r *Replicator) SequentialAck(key string, f ReplicatedAckFunc) error {
-	var e error
+	var me *multierror.Error
```

Comment:
Super nit: I think merr is a better name for multierrors. It's what they use in the docs also.

Reply:
Ah fair enough - guess that better matches the classic `err`!